### DISCUSSION TITLE
Asciify List Entries

### DISCUSSION THREAD

**Discussion by kontell** [Flexget/Flexget#3968] [Q&A]
I have a list from trakt which contains Shōgun, for it to successfully match it needs to be Shogun. Is there a way to convert the list?

This ability appears to be present in Jinja, but I've no idea how to use it.

https://flexget.com/Jinja

**Comment by BrutuZ** [comment]
Depends on how you use that list.
If you're using the `trakt-list` directly as a filter for another input plugin in the same task, then you can't, there's nowhere to edit the title.
If you have two separate tasks, then it is quite easy and actually recommended as it serves as a cache from when Trakt is having connection issues. On the first task you feed the `trakt-list` to an `entry-list`, there you can use `set` to asciify the title. On the second task you replace the `trakt-list´ input with the `entry-list` from the first task
